board/piksiv3/rootfs-overlay/etc/init.d/copy_sys_logs.sh

```diff
@@ -0,0 +1,58 @@
+#!/bin/ash
+
+log_tag=copy_sys_logs
+
+source /etc/init.d/sdcard.sh
+source /etc/init.d/logging.sh
+
+should_run()
+{
+  if needs_migration $MOUNTNAME; then
+    logw --sbp "Exiting: the SD card needs to be migrated..."
+    return 1
+  fi
+
+  if [[ -n "${COPY_SYS_LOGS}" ]]; then
```

Comment:
Testing this and this check does not seem to do what it aims. If "copy_system_logs=True" is not in my config.ini and I set "copy_system_logs" to True dynamically, the script still exits with "Exiting: logging is not enabled..."